remo/profiles/tasks.py

```diff
@@ -108,3 +114,21 @@ def set_unavailability_flag():
     (UserStatus.objects.filter(start_date__range=[get_date(-1), get_date()],
                                is_unavailable=False)
                        .update(is_unavailable=True))
+
+
+@periodic_task(run_every=timedelta(hours=24))
+def resolve_nomination_action_items():
+    """Resolve action items.
+
+    Resolve all the action items relevant to nomination reminders after the
+    10th day of each month."""
+
+    now_date = now().date()
+    if now_date.day == NOMINATION_END_DAY:
```

Comment:
You could replace this with ``now().date().day`` since you are not using now_date anywhere else.